alerting/temporal/src/qdrant/index.ts

```diff
@@ -7,22 +7,26 @@ import {
   GAUGE,
 } from "@datadog/datadog-api-client/dist/packages/datadog-api-client-v2/models/MetricIntakeType";
 
-const { QDRANT_CLUSTERS, QDRANT_MONITORING_API_KEY } = process.env;
+const { QDRANT_NODES, QDRANT_MONITORING_API_KEY } = process.env;
```

Comment:
Can we use the new way of getting the configuration?

Reply:
Fair ... we don't have the `types` module in this deployment and I did not want to add it for this, WDYT?